web/public/locales/da.json

```diff
@@ -19,7 +19,14 @@
     "electricity": "Elektricitet",
     "electricityproduction": "Elproduktion",
     "electricityconsumption": "Elforbrug",
-    "bysource": "efter kilde",
+    "by-source": {
+      "emissions": "CO₂-udledning per kilde",
```

Comment:
Do we need to differentiate between production and consumption here?

Reply:
Hmm, we currently don't and I don't think we necessarily need to either - "carbon emissions by source" kinda works either way :)